Library/Formula/pacvim.rb

```diff
@@ -0,0 +1,19 @@
+class Pacvim < Formula
+  homepage "https://github.com/jmoon018/PacVim"
+  url "https://github.com/jmoon018/PacVim/archive/v1.1.0.tar.gz"
+  sha1 "8393cf792b72ad77a7d8f6cdb691d30506af18c3"
+  head "https://github.com/jmoon018/PacVim.git"
+
+  # upstream bug report https://github.com/jmoon018/PacVim/pull/16
+  patch do
+    url "https://github.com/jmoon018/PacVim/pull/16.diff"
+    sha1 "8eb488866e20cd46e570f88da260b12b4681acca"
+  end
+
+  def install
+    inreplace "src/game.cpp", "maps/map", "#{libexec}/maps/map"
```

Comment:
Can you please submit this as a patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!